cluster_stub_test.go

```diff
@@ -329,6 +329,28 @@ func (scl *clusterStub) newCluster(opts ...ClusterOpt) *Cluster {
 	return c
 }
 
+func (scl *clusterStub) clientInitSyncFailedFunc(failedAddrsMap map[string]bool) ClientFunc {
+	return func(network, addr string) (Client, error) {
+		for _, s := range scl.stubs {
+			if s.addr == addr {
+				for failedAddr, flag := range failedAddrsMap {
```

Comment:
This would be better written as:

```go
if failedAddrsMap[s.addr] {
    failedAddrsMap[s.addr] = false
    return nil, errors.New("timeout")
}
```